### PR TITLE
add code for project 3

### PR DESCRIPTION
**Pull request by ns-26**

### CODE REVIEW

File: Project-3/NavdeepSingh/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Weather Application
```
Comment (ankurdubey521) [inline Project-3/NavdeepSingh/README.md:1]: Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!

Comment (ns-26) [reply Project-3/NavdeepSingh/README.md:1]: My code has an api key so is it ok to put the code publicly on github

Comment (ankurdubey521) [reply Project-3/NavdeepSingh/README.md:1]: @ns-26 are you paying for the API key? If not you can host it publicly for now

Comment (ns-26) [reply Project-3/NavdeepSingh/README.md:1]: done.....do i need to create a new PR

Comment (ankurdubey521) [reply Project-3/NavdeepSingh/README.md:1]: @ns-26 you can push a commit to the same branch, it will show up below

**Pull request merged**